FinSearchUnified/Tests/BusinessLogic/ExportTest.php

```diff
@@ -49,6 +57,26 @@ protected function tearDown()
         Utility::sResetArticles();
     }
 
+    /**
+     * @throws \Doctrine\ORM\OptimisticLockException
+     */
+    public static function tearDownAfterClass()
+    {
+        parent::tearDownAfterClass();
+
+        // Reset category active status which was used in the test cases
+        $categoryRepository = Shopware()->Models()->getRepository('Shopware\Models\Category\Category');
+
+        $categoryModels = $categoryRepository->findBy(['id' => [Shopware()->Shop()->getCategory()->getId(), 39]]);
```

Comment:
Use `$this->categoryRepository` instead instead.

Reply:
It is called inside a `static` method so $this wont be available

Reply:
You are absolutely right.